smswebapp/settings/docker.py

```diff
@@ -13,7 +13,7 @@
 
 # Redirect all traffic except healthz endpoint to HTTPS unless DANGEROUS_DISABLE_HTTPS_REDIRECT
 # is set
-SECURE_SSL_REDIRECT = os.environ.get('DANGEROUS_DISABLE_HTTPS_REDIRECT') is not None
+SECURE_SSL_REDIRECT = os.environ.get('DANGEROUS_DISABLE_HTTPS_REDIRECT') is None
```

Comment:
Perhaps this should be set in the development and production docker-compose configs since redirecting to https://localhost:8000/ makes little sense?